backoffice/src/Models/FormModels/LetterForm.cs

```diff
@@ -39,8 +49,18 @@ public LetterForm(Letter letter)
     {
         return new Dictionary<string, dynamic>
         {
+            { "email address", Recipient},
```

Comment:
This is great for now, but thinking of re-working this, but not for right now. 

If we have a set list of variables that can be used, I wonder if it's better to have these in a LetterVariables obj or something instead of dynamic, then something in that object that builds this Dictionary<string, dynamic> for the email client to use